src/app/contribute/guidelines/components/index.html

```diff
@@ -0,0 +1,201 @@
+    </li>
+    <ul>
+      <li>
+        It allows the SKY UX team to review your code in manageable pieces. It's much easier to review 3 pull requests with 50 files than 1 pull request with 150 files.
+      </li>
+      <li>
+        It allows you to get feedback sooner. Catching pervasive coding style changes in a small code review allows you update the rest of your code before you submit it.
+      </li>
+      <li>
+        It demonstrates that your code is modular and does not contain excessive interdependencies. For instance, if Component B is a child component of Component A, you should be able to author, test, document, review, and release Component B in the absence of Component A. If your changes must include both Component A and Component B to be tested and reviewed, then it raises a red flag that the components may be too tightly coupled and should be refactored.
+      </li>
+    </ul>
+    <li>
+      <b>If you make an API change that affects inputs and outputs for a component, we recommend that you provide your potential (not working) documentation and demo to the SKY UX team for review.</b> This lets us head off API issues before implementation. Contributors should keep in mind that the API can change in the course of development.
+    </li>
+  </ul>
+
+  <stache-page-anchor>
+    Localization
+  </stache-page-anchor>
+
+  <p>
+    To declare localization strings, specify a string name and provide the string and a description in the <stache-code>resources_en_US.json</stache-code> file in <stache-code>src/assets/locales</stache-code>. You can reference the localization strings with the <stache-code>skyLibResources</stache-code> pipe or the <stache-code>SkyLibResourcesService</stache-code> service in <stache-code>@skyux/i18n</stache-code>.
+    </p>
+
+  <stache-page-anchor>
+    Get the code
+  </stache-page-anchor>
+
+  <ol>
+    <li>
+      <a href="https://help.github.com/articles/fork-a-repo/">Fork the repo</a> into your own GitHub repo.
+    </li>
+    <li>
+      Create a branch off of master named after the feature you will contribute. The branch name should be kebab-case (my-new-feature).
+    </li>
+    <li>
+      Clone your repo locally and run <stache-code>nvm use</stache-code> to ensure you have the preferred version of Node.js installed. Next, run <stache-code>npm install</stache-code> to install all required dependencies. For more information on prerequisites, see <a stacheRouterLink="/learn/get-started/prereqs">SKY UX prerequisites</a>.
+    </li>
+    <li>
+      <a stacheRouterLink="." fragment="write-visual-regression-tests"></a>Run your first round of visual regression tests with <stache-code>skyux e2e</stache-code></a>. This establishes baseline screenshots of the components in the master repo so that as you make changes and run your visual regression tests, your changes are validated against the state of the repo before the changes.
```

Comment:
The link here isn't working. We need to move the closing `a` tag to the end of the first sentence.